src/configs/constants.ts

```diff
@@ -0,0 +1,26 @@
+export const Constants = {
+  LONG_DATE_FORMAT: "DD/MM/YYYY",
+  BRAND_COLORS: [
+    "#4cebb5",
+    "#bcdd99",
+    "#1f2d84",
+    "#6700d8",
+    "#24ad5b",
+    "#a5d7fd",
+    "#fdc18b",
+    "#c4a4e1",
+    "#00b3e8",
+    "#ea6f9b",
+    "#ff7e7e",
+    "#c28686",
+    "#ff8f1d",
+    "#97ce84",
+  ],
+  APIS: {
+    SEARCH_WATCHES: "searchWatches",
```

Comment:
I would not store every endpoint path inside a separate variable. Also endpoint paths should follow the *kebab-case* convention.